### DISCUSSION TITLE
The email round up sent on 1 May advertises events for June

### DISCUSSION THREAD

**Discussion by Arielle-Bennett** [openresearchcalendar/Open-Research-Calendar#52] [Ideas]
Hi there! 

The email round up from the Open Research Calendar sent on 1 May advertises events for June. I would suggest it's more helpful for people to see events advertised for the same month (e.g. May events sent on 1 May, June events sent 1 June). 

What do others think? When is most helpful to have events advertised to you? 

Arielle

**Comment by b-kennedy0** [comment]
Hi @Arielle-Bennett, Thank you so much for the suggestion. I completely understand what you are suggesting. Initially, we thought that sending out events of the same month might not have given people enough time to plan attendance/source any funding etc. However, I can see merit in also including the current month. 

The team have begun some discussions about how best to proceed. We are now thinking perhaps the current month & next month together. E.g. May email would contain May + June. Then June email would contain June + July. This could allow people to see any events that had been added since the last email. 

We'll keep this discussion open to find out what works best! Thanks again 😊

**Comment by Arielle-Bennett** [comment]
Thanks for the response! I think the solution you've suggested will work well and allow for updates closer to events. Look forward to my June email 😄
